### PR TITLE
Update wallet connect sign to 1.0.2 version #5693

### LINKED ISSUE #5693: Update WalletConnect Sign v2 SDK

### PR DESCRIPTION
**Pull request by oa-s**
Closes #5693

### CODE REVIEW

File: AlphaWallet.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -9,6 +9,15 @@
         "version" : "0.20200225.4"
       }
     },
+    {
+      "identity" : "bigint",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/attaswift/BigInt.git",
+      "state" : {
+        "revision" : "0ed110f7555c34ff468e72e1686e59721f2b0da6",
+        "version" : "5.3.0"
```
Comment (hboon) [inline AlphaWallet.xcworkspace/xcshareddata/swiftpm/Package.resolved:18]: We are using ~> v3.1 in our `Podfile` though. Any issue?

**Comment by oa-s** [comment]
have no idea, example app use it, will try with latest one

**Review by hboon — APPROVED** [review]

**Comment by oa-s** [comment]
tests crushes due to conflicting bettwen `BigInt.BigInt` libs, one from spm and one from Cocoa Pods

**Comment by hboon** [comment]
Yeah, I thought 2 different versions might be a problem.

**Review by hboon — APPROVED** [review]
Tests passed. How did we get pass the BigInt issue?

**Comment by oa-s** [comment]
> Tests passed. How did we get past the BigInt issue?

pod resolves this issue, use bigInt from pod only

**Comment by oa-s** [comment]
@hboon i was trying to test with dapp, it doesn't work, stucks on pairing,  might be something with api key

**Comment by oa-s** [comment]
@hboon can u try it on your side?

**Comment by oa-s** [comment]
merging, and create a new issue `verify wallet connect v2 connection` facing the same with example app

**Comment by hboon** [comment]
For posterity, the issue: #5983

**Pull request merged**